api-channel-issue-tracker/src/main/java/com/synopsys/integration/alert/api/channel/issue/search/IssueCategoryRetriever.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * api-channel-issue-tracker
+ *
+ * Copyright (c) 2021 Synopsys, Inc.
+ *
+ * Use subject to the terms and conditions of the Synopsys End User Software License and Maintenance Agreement. All rights reserved worldwide.
+ */
+package com.synopsys.integration.alert.api.channel.issue.search;
+
+import java.util.function.BooleanSupplier;
+
+import com.synopsys.integration.alert.api.channel.issue.model.ProjectIssueModel;
+import com.synopsys.integration.alert.api.channel.issue.search.enumeration.IssueCategory;
+import com.synopsys.integration.alert.processor.api.extract.model.project.ComponentConcernType;
+
+public class IssueCategoryRetriever {
```

Comment:
This class can be a stateless `@Component` as outlined by the Alert Team Preferences page.

Reply:
That also means the methods should not be static.